### DISCUSSION TITLE
Add a prop to the Transfer component to allow for a custom icon for the dropdown in the title

### DISCUSSION THREAD

**Discussion by stefenmenzel** [ant-design/ant-design#43770] [Ideas]
The antd transfer component has an icon in the header and I would like to switch that icon without losing all the built in functionality and having to re-write it.

![image](https://github.com/ant-design/ant-design/assets/45863897/9704c2ec-2178-41cd-aa8b-15ffe78696c2)

I would like to change that down arrow to a different icon via a prop.

Is it possible to get a change like this implemented in a future version?

**Comment by li-jia-nan** [comment]
What do you expect the API to look like ?

 be like:

```tsx
<Transfer
  dataSource={[]}
  leftDropdownIcon={<div>left</div>}
  rightDropdownIcon={<div>right</div>}
/>
```

or:

```tsx
<Transfer
  dataSource={[]}
  dropdownIcon={{
    left: <div>left</div>,
    right: <div>right</div>,
  }}
/>
```

**Reply by stefenmenzel** [reply]
I like the top option, I would mostly like an option for the dropdown button in the title, the one that points down. But it would be GREAT to have a handle on all three.

```
<Transfer
    dataSource={[]}
    leftDropdownIcon={<div>left</div>}
    rightDropdownIcon={<div>right</div>}
    titleDropdownIcon={<div>title</div>}
/>
```

**Reply by MadCcc** [reply]
Simply one prop is enough.

**Comment by li-jia-nan** [comment]
- #43773

**Reply by stefenmenzel** [reply]
That's exciting! When will this be available in the NPM package?

**Reply by li-jia-nan** [reply]
about a week later
